Tests/SwiftlyTests/SwiftlyTests.swift

```diff
@@ -461,4 +535,67 @@ public struct MockToolchainDownloader: HTTPRequestExecutor {
 
         return try Data(contentsOf: archive)
     }
+
+#elseif os(macOS)
```

Comment:
The first two halves of this branch are identical, can we combine them and only have the OS-specific parts inside the branches?